frontend/src/Pages/AuthenticationPage/LoginPage.jsx

```diff
@@ -73,6 +74,11 @@ export function LoginPage() {
                 required: true,
                 message: PASSWORD_REQUIRED,
               },
+              {
+                pattern:
+                  /^(?=.*[A-Za-z0-9])[A-Za-z0-9!*_@#$%^&+=<>|.,:;"'{})(-?/`~]*$/,
+                message: PASSWORD_ONLY_ALLOW,
```

Comment:
Redudant comma

Reply:
Redundant comma is auto generated by Format code Prettier